include/cuco/dynamic_map.cuh

```diff
@@ -101,12 +101,12 @@ class dynamic_map {
   static_assert(std::is_arithmetic<Key>::value, "Unsupported, non-arithmetic key type.");
 
  public:
-  using value_type      = cuco::pair_type<Key, Value>;       ///< Type of key/value pairs
-  using key_type        = Key;                               ///< Key type
-  using mapped_type     = Value;                             ///< Type of mapped values
-  using atomic_ctr_type = cuda::atomic<std::size_t, Scope>;  ///< Atomic counter type
+  using value_type      = cuco::pair_type<Key, Value>;            ///< Type of key/value pairs
+  using key_type        = Key;                                    ///< Key type
+  using mapped_type     = Value;                                  ///< Type of mapped values
+  using atomic_ctr_type = cuda::atomic<std::size_t, Scope>;       ///< Atomic counter type
```

Comment:
this is odd too

Reply:
This is the reason I'm reluctant to bump clang-format version since it requires us to update the config file frequently to preserve consistent formatting rules.